magento1-module/app/code/local/Divante/VueStorefrontBridge/Helper/Address.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+class Divante_VueStorefrontBridge_Helper_Address extends Mage_Core_Helper_Abstract
+{
+    /**
+     * @param array $streetData
+     *
+     * @return string
+     */
+    public function concatStreetData(array $streetData): string
+    {
+        $concatStreetData = '';
+
+        if (isset($streetData[0], $streetData[1])) {
+            $concatStreetData = $streetData[0] . ' ' . $streetData[1];
```

Comment:
Magento stores address separated by a new line. This prevents false splits when a street contains a space in the name itself. Might I suggest `implode("\r\n", $streetData);` here?